Makefile

```diff
@@ -23,6 +23,6 @@ $(BUILD)/uf2_version.h: Makefile
 
 OBJ += $(BUILD)/uf2_version.h
 
-include $(TOP)/hw/chip/$(CHIP_FAMILY)/$(CHIP_VARIANT).mk
+include $(TOP)/hw/chip/$(TUF2_CHIP_FAMILY)/$(TUF2_CHIP_MEMBER)/$(TUF2_CHIP_VARIANT).mk
```

Comment:
`CHIP_FAMILY` is used by tinyUSB, is there any benefit from renaming it?

Reply:
Only because the other two elements have the TUF2 on the front, showing they're 'owned' by this code...if we adopt a 'style' to make it easier for the reader, then we shouldn't break it on the very same line :-)  CHIP_FAMILY is something different and receives the value 'transdimension' which means there had to be an absolute path in family.mk otherwise it picks up the wrong value. This approach avoids that and allows things to be more standard.

Reply:
Ok, I'm convinced :)